internal/config/seccomp/seccomp.go

```diff
@@ -194,11 +194,6 @@ func (c *Config) setupFromField(
 	log.Debugf(ctx, "Setup seccomp from profile field: %+v", profileField)
 
 	if c.IsDisabled() {
-		if profileField.ProfileType != types.SecurityProfileTypeUnconfined {
-			return errors.Errorf(
-				"seccomp is not enabled, cannot run with a profile",
-			)
-		}
```

Comment:
I don't think that we have to be too strict in case seccomp is disabled and can silently fail as proposed. WDYT @haircommander ?

Reply:
I am hesitant.

On the one hand, I've personally hit this, and the only way I could figure out how to fix it (in the crunch time before a demo) was hard-code this check away. 
On the other hand, it's hard to tell if this is accurately hitting expectation. It feels like a user who doesn't want seccomp should change the kubelet default profile to be unconfined. If we silently don't respect the security settings, that sounds like a CVE in the making (in a day where we assume it's set but isnt' and we don't hear about it until a CVE is reported).

At the very least, we should logrus.Errorf() here to scream to the admin that they're not actually running with seccomp

Reply:
I concur that we should at the very least leave behind a error/warning message here